src/main/resources/application.conf

```diff
@@ -1,6 +1,6 @@
 mantis {
-  # Identifier used when connecting to other clients
-  client-id = "mantis"
+  # Optionally override the client ID sent in Hello messages.
+  client-id = null
```

Comment:
What would be the use case for an user configuring this manually? Why not using only the generated in VersionInfo?

Reply:
I have no idea, I just didn't want to remove it since the current "mantis" value can also be overwritten. I thought even that maybe the first part of the name should come from here, not from the project name. I think Besu supports some kind of [identity](https://github.com/hyperledger/besu/blob/c8e23c970ebef4297d6e75abec784f7024df340c/besu/src/main/java/org/hyperledger/besu/BesuInfo.java#L33) but it's unclear what that might be. 

Anyway it's up to you, we can remove it completely if you prefer.

Reply:
Changed it so it's more like besu: if you set it it will be appended to the `mantis` part, so we can distinguish it like `mantis-etc` or whatever.

Reply:
I don't like much the name of this config, the client id would be after the augmentation, not the augmentation alone... maybe something like `client-id-augmentation`?

Reply:
That's a bit of a mouthful :sweat_smile: 
Should we name it `client-identity` after the besu setting that inspired it?

Reply:
Renamed, and also changed the format slightly to match besu, based on their [tests](https://github.com/hyperledger/besu/blob/72e975e8e7e415fc396bf26c515fc0f3e8d23fba/besu/src/test/java/org/hyperledger/besu/BesuInfoTest.java#L48-L54) explaining this is the ethstats friendly way.

Reply:
Should we maybe copy those tests?

Reply:
I thought [this](https://github.com/input-output-hk/mantis/pull/794/files#diff-a00369dbb33d886362f525713ce1b0f7e1ca5a14151127b91ff44c5864e1dc67R10) test captures the gist of it.